### DISCUSSION TITLE
Colouring

### DISCUSSION THREAD

**Discussion by prashjet** [dynamics-of-stellar-systems/dynamite#403] [Announcements]
Thomas and I have pencilled in Wednesday 31st 14:00 for the next colouring meeting. Please respond with a positive emoji if you are able to attend, or suggest an alternative time/date otherwise.

Prash

**Comment by prashjet** [comment]
Hi all. Reminder: we'll meet today at 14:00.

**Comment by boeckera** [comment]
Sorry I totally missed this :(

**Comment by prashjet** [comment]
No worries! Some brief minutes:

- we're making progress on reading in the population data: #398 (Thomas tying up loose ends, Prash to test)
- we're ready for some realistic test data! @liquidreams: can you provide us a test galaxy? i.e.
   - pick one of your galaxies with a good fit to the kinematics
   - to keep this separate from your ongoing work, I suggest you copy the main directory for this galaxy to a new location on VSC where we all have permission to modify files. We will use that for development
   - add a table with populaon data: age, d_age, metal, d_metal
- the next step is to bin the orbit distributions. @boeckera: can you arrange the next meeting at least where we can start to discuss this?
- @afeldmei will try our testing and merging #404

**Reply by boeckera** [reply]
the next regular dynamite meeting is planned for next week friday, would that work?

**Comment by liquidreams** [comment]
Dear all, Prash,

The MCMC is (I hope) currently running so that we have realistic errors for the age and metallicity estimates. I'll be in vacation the next 4 days but I'll anyway check and once it's done I'll try to send you what you require asap.

Thank you for this :)
Kind regards,
Iris

⁣--
Iris Breda
Marie Curie postdoctoral fellow
Institut für Astrophysik @ UniVie, Vienna, Austria

Get BlueMail for Android ​

**Comment by prashjet** [comment]
Hi all,

@liquidreams has provided luminosity weighted age and metallicity maps for NGC1042, with MCMC uncertainties. Thanks you Iris :)

Data file: [NGC1042_ageLZL.txt](https://github.com/user-attachments/files/16495474/NGC1042_ageLZL.txt) note: I changed the file suffix from `.ecsv` to `.txt` in order to upload to GitHub, but `dynamite` expects the former.

Iris adds:

> Here it follows the values for the age and Z for NGC1042 (lum-weig.).
> You dont need the mass estimates right? I've been using FADO's mass
> estimates to constrain the ML ratio, maybe its an idea for the future ;)
> In addition, I send you an image with the maps given by FADO, to have an
> idea of how these quantities vary.
> Let me know if you need something else.
 
![NGC1042_FADO_maps_BC03](https://github.com/user-attachments/assets/c15b859a-17aa-45f0-8f96-88a2660351cd)

**Comment by prashjet** [comment]
Iris's test data:

/gpfs/data/fs71474/iris/sample/orb_col/

**Comment by maindlt** [comment]
Quick update on Voronoi binning / tessellation of the circularity plot data:
After a little survey I think that `scipy.spatial.Voronoi()` cannot be used out of the box (it does the tessellation based on nearest-neighbor map). Instead, `vorbin` (https://pypi.org/project/vorbin/) with `target_sn` set to the desired weight in each bin and the optional keyword `sn_func` defined such that it returns the weight looks more promising...

**Comment by boeckera** [comment]
I am dumbing here useful/necessary papers for implementing the orbit coloring:

**Method Paper and FCC167 papers from Ling:**
[https://ui.adsabs.harvard.edu/abs/2020MNRAS.496.1579Z/abstract](https://ui.adsabs.harvard.edu/abs/2020MNRAS.496.1579Z/abstract)
[https://ui.adsabs.harvard.edu/abs/2022A%26A...664A.115Z/abstract](https://ui.adsabs.harvard.edu/abs/2022A%26A...664A.115Z/abstract)

**Orbit coloring of all Fornax3D galaxies:**
[https://ui.adsabs.harvard.edu/abs/2023A%26A...672A..84D/abstract](https://ui.adsabs.harvard.edu/abs/2023A%26A...672A..84D/abstract)

**Orbit coloring of CALIFA galaxies:**
[https://ui.adsabs.harvard.edu/abs/2024A%26A...681A..95J/abstract](https://ui.adsabs.harvard.edu/abs/2024A%26A...681A..95J/abstract)

Please feel free to add to this and any other paper that might be of relevance for a future publication!

Cheers!
